src/conv/k-g/region_list.cpp

```diff
@@ -65,22 +65,19 @@ static void  writeAttributes
 ) {
 
     struct rt_db_internal     region_internal;
-    struct directory*         dp;
-    struct db_i*              dbip;
-    bu_attribute_value_set*   avs;
 
-    dbip = wdbp->dbip;
-    dp = db_lookup(dbip, name, 0);
+    struct db_i*      dbip = wdbp->dbip;
+    struct directory* dp = db_lookup(dbip, name, 0);
 
-    rt_db_get_internal(&region_internal, dp, dbip, bn_mat_identity, &rt_uniresource);
-    avs = &region_internal.idb_avs;
+    rt_db_get_internal(&region_internal, dp, dbip, NULL, &rt_uniresource);

```

Comment:
do you think that it's neccessary to check the id returend by 
rt_db_get_internal()==ID_COMBINATION?

Reply:
That's a good point. However, the ID is not so important, because *writeAttributes()* could write the attributes to any database object. But, you should check if the command was successful, i.e. thje retzurn is >= 0.

Same for *dp*?

Reply:
they both need testing. I modified the code. please let me know if you have any notes